backend/kesaseteli/kesaseteli/settings.py

```diff
@@ -94,6 +95,7 @@
 STATIC_ROOT = env("STATIC_ROOT")
 MEDIA_URL = env.str("MEDIA_URL")
 STATIC_URL = env.str("STATIC_URL")
+EXCEL_ROOT = os.path.join(MEDIA_ROOT, "excel")
```

Comment:
Good point! As this is supposed to be a low effort solution, I decided to leave out the feature of keeping the files on server. I will add it later if it is needed but according to a chat with the POs this might not be needed.